docs/organizations/managing-security-and-risk.md

```diff
@@ -3,9 +3,9 @@
 !!! info "This is a preview feature"
     This is a new Codacy feature and <span class="skip-vale">we're</span> continuing to improve it.
 
-The security and risk management feature helps you <span class="skip-vale">quickly</span> identify, track, and address security issues by automatically opening time-bound, prioritized action items whenever Codacy detects security issues in your organization repositories or in your connected Jira Cloud instance.
+The Security and risk management feature helps you <span class="skip-vale">quickly</span> identify, track, and address security issues by automatically opening time-bound, prioritized action items whenever Codacy detects security issues in your organization repositories or in your [connected Jira instance](./integrations/jira-integration.md).
```

Comment:
It's capitalized "Security and risk" in the UI and we generally use sentence case (capitalize only the first word).

I personally prefer title case as you suggest, since it clarifies all the words that make up the feature, but we'll have to go with sentence case here for consistency.

```suggestion
The Security and risk management (SRM) feature helps you <span class="skip-vale">quickly</span> identify, track, and address security issues by automatically opening time-bound, prioritized action items whenever Codacy detects security issues in your organization repositories or in your [connected Jira instance](./integrations/jira-integration.md).
```

Reply:
Makes sense to me!